.node-version

```diff
@@ -1 +1 @@
-16
+18
```

Comment:
do we need to update any documentation with new node version? FURN ok?

Reply:
It will have to be part of the overall downstream integration. FURN actually has had trouble updating node.

Reply:
We filter ReactAndroid on lines 25-44. So I think this is fine.